client/disputes/index.tsx

```diff
@@ -99,55 +103,72 @@ const headers = [
 	},
 ];
 
-export const DisputesList = () => {
+export const DisputesList = (): JSX.Element => {
+	/* eslint-disable */
 	const { disputes, isLoading } = useDisputes( getQuery() );
 
+	const {
+		disputesSummary,
+		isLoading: isSummaryLoading,
+	} = useDisputesSummary();
+
 	const rows = disputes.map( ( dispute ) => {
-		const order = dispute.order
-			? {
-					value: dispute.order.number,
-					display: <OrderLink order={ dispute.order } />,
-			  }
-			: null;
+		const {
+			dispute_id,
```

Comment:
we could use the `dispute` object instead of destructuring it.

Reply:
Happy to change it. It is true that the destructuring assingment is really popular nowadays in the JS world, maybe  too much :)

Reply:
I think the point about destructuring being memory inefficient is arguable as only a handful of properties are being destructured here. That said, I am in favor of accessing `dispute`s props directly via dot notation for readability.

> It is true that the destructuring assingment is really popular nowadays in the JS world, maybe too much :)

Haha. To a point that a lot of devs no longer care about readability. I once saw the destructuring of 35 or 40 props in a `render` method long enough to fill 20 pages of A4 sheets 😛 After a short while, I lost track of what variable was local and which ones were props.

Reply:
Done! :)